doc/sphinx/language/gallina-extensions.rst

```diff
@@ -157,9 +155,9 @@ It can be activated for printing with
 .. _record_projections_grammar:
 
   .. productionlist:: terms
-     term : term `.` ( qualid )
-          : | term `.` ( qualid arg … arg )
-          : | term `.` ( @`qualid` `term` … `term` )
+     projection : projection `.` ( `qualid` )
+          : | projection `.` ( `qualid` `arg` … `arg` )
+          : | projection `.` ( @`qualid` `term` … `term` )
```

Comment:
Same remark. I think `term` was really meant but that should have been an extension. I also think that this should be part of the main grammar in the first chapter.

Reply:
I added FIXMEs